charts/kong/values.yaml

```diff
@@ -963,13 +990,18 @@ podSecurityPolicy:
 priorityClassName: ""
 
 # securityContext for Kong pods.
-securityContext: {}
+securityContext:
+  runAsUser: 1000
+  runAsGroup: 1000
```

Comment:
```suggestion
```
These (and whether setting them actually breaks anything) will be very image-specific, so I don't think we should be setting them here.

We could set `runAsNonRoot: true` at the Pod level, which will cause Kubernetes to reject the Pod if any additional containers aren't set to or built with non-0 UIDs.

If the linter's too stupid to let that pass the rule instead, then yeah, the rule's a bit busted and we should just leave it off.

Reply:
Seems to work with only setting these in the containerSecurityContext